Utilities/Doxygen/DoxygenConfig.cmake

```diff
@@ -89,52 +89,52 @@ set(DOXYGEN_PREDEFINED
+    "itkGetVector3Macro(name,type)=                   virtual type* Get##name () const;                   virtual void Get##name (type& _arg1, type& _arg2, type& _arg3) const;                   virtual void Get##name (type _arg[3]) const;"
+    "itkSetVector4Macro(name,type)=                   virtual void Set##name (type _arg1, type _arg2, type _arg3, type _arg4)                   virtual void Set##name (type _arg[4]);"
+    "itkGetVector4Macro(name,type)=                   virtual type* Get##name () const;                   virtual void Get##name (type& _arg1, type& _arg2, type& _arg3, type& _arg4) const;                   virtual void Get##name (type _arg[4]) const;"
+    "itkSetVector6Macro(name,type)=                   virtual void Set##name (type _arg1, type _arg2, type _arg3, type _arg4, type _arg5, type _arg6)                   virtual void Set##name (type _arg[6]);"
+    "itkGetVector6Macro(name,type)=                   virtual type* Get##name () const;                   virtual void Get##name (type& _arg1, type& _arg2, type& _arg3, type& _arg4, type& _arg5, type& _arg6) const;                   virtual void Get##name (type _arg[6]) const;"
+    "itkSetVectorMacro(name,type,count)=                   virtual void Set##name(type data[]);"
+    "itkGetVectorMacro(name,type,count)=                   virtual type* Get##name () const;"
+    "itkSetInputMacro(name,type)=         virtual void Set##name(const type *input);"
+    "itkGetInputMacro(name,type)=         virtual const type * Get##name() const;"
+    "itkNewMacro(type)=                   static Pointer New();                                                                           virtual::itk::LightObject::Pointer CreateAnother() const;"
+    "itkSimpleNewMacro(type)=             static Pointer New();"
+    "itkCreateAnotherMacro(type)=         virtual::itk::LightObject::Pointer CreateAnother() const;"
+    "itkFactorylessNewMacro(x)=                   static Pointer New();                   virtual ::itk::LightObject::Pointer CreateAnother() const;"
+    "itkTypeMacro(thisClass,superclass)=                   virtual const char *GetNameOfClass() const;"
+    "itkVirtualGetNameOfClassMacro(thisClass)= virtual const char * GetNameOfClass() const;"
+    "itkOverrideGetNameOfClassMacro(thisClass)= const char * GetNameOfClass() const override;"
+    "itkEventMacro(thisClass,superclass)=                   class thisClass : public superclass {};"
+    "itkDeclareExceptionMacro(newexcp,parentexcp,whatmessage)=         namespace itk {         class newexcp : public parentexcp {        public:       newexcp(const char *file, unsigned int lineNumber) :        parentexcp(file, lineNumber)          { this->SetDescription(whatmessage); }          newexcp(const std::string & file, unsigned int lineNumber) :          parentexcp(file, lineNumber)          { this->SetDescription(whatmessage); }           itkOverrideGetNameOfClassMacro(newexcp, );         };        }"
```

Comment:
This might be the only change of substance (`itkTypeMacro(newexcp, , parentexcp)\;` into `itkOverrideGetNameOfClassMacro(newexcp, );`)